content/dev/index.md

```diff
@@ -117,11 +117,8 @@ Check the <a href="http://projects.apache.org/indexes/quick.html" target="_blank
 -  [Email etiquette tips for contributors](contrib-email-tips) 
 -  How to [moderate Apache mailing lists](committers.html#mail-moderate) 
 -  All Apache mailing lists are archived:
-    - Read most Apache [list archives on mail-archives.apache.org](http://mail-archives.apache.org/mod_mbox/) by going to the specific project list, then append the mbox filename to the URL (e.g. [mail-archives.apache.org/mod_mbox/httpd-dev/201210.mbox](mail-archives.apache.org/mod_mbox/httpd-dev/201210.mbox))
-    - Raw mbox archive files are available for every mailing list.
-    - Each project website has a "mail" directory, e.g. [httpd.apache.org/mail/](http://httpd.apache.org/mail/)
-(see [notes](/dev/project-site.html#mail)).
-    - For top-level/non-project mailing lists see [www.apache.org/mail/](/mail/) directory.
```

Comment:
https://www.apache.org/mail has been redirected to lists.a.o also.

Reply:
We previously have a link to `mail-archives.apache.org`. Do you mean we can use always `https://www.apache.org/mail`?

For example:

```
- For top-level/non-project mailing lists, see [list archives on www.apache.org/mail](/mail/).
```

Reply:
OK. Let me revert the changes on this file now and focus on the other changes.

I'll consider this part and may propose a new PR later.